src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -119,10 +118,16 @@
   (let [res (re-find re s)]
     (if (string? res) [res] res)))
 
-(defn- method-match?
-  "Return true if both mathod a and b match or if one is :any"
-  [a b]
-  (or (= a b) (some #{:any} [a b])))
+(schema/defn method-match?
+  "Return true if the provided method is equal to the value of `specified`. If
+  `specified` is a list of methods, return true if `method` is contained in
+  `specified`. If `specified` is set to :any then all methods will result in
+  true."
+  [method :- schema/Keyword
```

Comment:
Ring might hypothetically use an HTTP method for a request that we're not covering in our schema - e.g, PATCH or some user-defined method - so probably best not to restrict it here?